src/FrontDoor/FrontDoor.Test/ScenarioTests/FrontDoorTests.ps1

```diff
@@ -18,48 +18,76 @@ Full Front Door CRUD cycle
 #>
 function Test-FrontDoorCrud
```

Comment:
>Test-FrontDoorCrud [](start = 9, length = 18)

Add a test case Test-FrontDoorCrudDefaults passing only mandatory parameters and see if the defaults are populated correctly.